xml/GenCode.php

```diff
@@ -4,7 +4,8 @@
   die("GenCode can only be run from command line.");
 }
 
-ini_set('include_path', '.' . PATH_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'packages' . PATH_SEPARATOR . '..');
+$includes = ['.', '../packages', '../../civicrm-packages', '..'];
+ini_set('include_path', implode(PATH_SEPARATOR, $includes));
```

Comment:
The original mix of `PATH_SEPARATOR` and `DIRECTORY_SEPARATOR` and dots might make one's glaze over. (At least, it made my eyes glaze over...) Trying to summarize this bit, it:

* Switches to array notation to make it is easier to read
* Retains all the existing items in the search-list (`.`, `../packages`, `..`)
* Adds a new item to the search-list (`../../civicrm-packages`)